### ISSUE TITLE
run.properties file changes for CERA to post NAM-driven storm runs

### ISSUE THREAD

**Issue by carolakaiser** [jasonfleming/asgs#154]
The 2019 hurricane season produced a number of weak storms and the NAM-driven results provided good results. As a result, NAM driven storm runs are considered to get added to the storm ensemble as provided by the ASGS and displayed on CERA. To achieve this, a change in the run.properties file for NAM driven runs would be necessary to reflect the storm information. The goal is to post the full track information on the CERA website for such runs and also to bring them to the “Storm” tab.

The additional information in the run.properties file that CERA needs for such a run is as follows:

track_modified : n
storm class : HU
stormname : MICHAEL
track_raw_dat : bal142018.dat
track_raw_fst : al142018.fst
storm : 14
stormnumber : 14
advisory : 16
WindModel : WNAMAW12-NCP
stormname:MICHAEL
stormclass:HURRICANE
wind:125
advisory time: 1500 UTC WED OCT 10 2018

…

Everything else would stay unchanged.

**Comment by jasonfleming** [comment]
Yes, we can add metadata to the run.properties file as a post processing step for NAM runs that are intended to represent TCs. Beyond the CERA use case, I think this approach also enables archival/reuse value for our ADCIRC data because it facilitates search hits on NAM driven results when someone searches on TS NESTOR (for example). 

I would suggest one tweak based on (a) avoiding ambiguity regarding the type of forcing that was actually used in the run; and (b) my assumption of relative ease in modifying CERA code to include a new property that does the exact same thing (on the CERA side) as an existing property. 

My tweak to your suggestion is as follows: prepend the TC-associated properties in NAM runs with a "ref." prefix that indicates that the results have also been deemed valid for a particular TC by our Subject Matter Experts (SMEs). So for example, in a pure GAHM run we might have: 

stormnumber : 14

and the analogous metadata for a NAM run would be

ref.stormnumber : 14

The automated addition of this metadata would be configured on a case-by-case basis according to the opinions of the SMEs regarding the character of the storm, the mesh, geographic location of the storm track, etc. 

My assumption behind this tweak is that it would be relatively easy to implement in CERA by simply changing instances of e.g. "if ( stormnumber )..." to "if ( stormnumber || ref.stormnumber ) ..." in your python code. Not sure whether this is true. In any case. please let me know whether this tweaked approach would work for you.

**Comment by jasonfleming** [comment]
(From Carola)

As far as I understand it now, putting the NHC track on a  NAM run might not be the best idea, so I would like to leave the functionality as we have it right now (just showing a thin track line with the hurricane symbol and the storm name.

However, having the storm information in the run.properties file is still very valuable because it would allow me to better address this information in the map header and to “mark” those runs in the CERA database so that they do not get deleted.

Jason, I like your idea of having a  special designation and would like to give this a try. Could you please make sure that all storm related info in the run.properties file would get the .ref name? Otherwise, CERA would probably go into some trouble.

**Comment by wwlwpd** [comment]
Seems related to #271
